src/custom-types.json

```diff
@@ -688,6 +688,15 @@
       "https://tools.ietf.org/html/rfc7763"
     ]
   },
+  "text/mdx": {
+    "compressible": false,
+    "extensions": ["mdx"],
+    "notes": "JSX in Markdown",
+    "sources": [
+      "https://spectrum.chat/thread/1021be59-2738-4511-aceb-c66921050b9a",
+      "https://github.com/mdx-js/specification"
```

Comment:
Please provide a primary source to where this mime is defined. https://github.com/jshttp/mime-db/blob/master/README.md#contributing

I don't see ant mention of the mime type on this page. Perhaps I'm missing it though, if you can point me to were it states the mime and that this file extension it meant to contain the contents for this mime.